src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py

```diff
@@ -0,0 +1,156 @@
+from base64 import b64encode
+import requests
+from requests.utils import to_native_string
+
+from azure.cli.core.prompting import prompt, prompt_pass, NoTTYException, prompt_y_n
+import azure.cli.core.azlogging as azlogging
+from azure.cli.core.util import CLIError
+
+from ._utils import (
+    get_resource_group_name_by_registry_name,
+    get_registry_by_name
+)
+from .credential import acr_credential_show
+
+logger = azlogging.get_az_logger(__name__)
+APIUSERNAME = ""
+APIPASSWORD = ""
+
+class NotFound(Exception):
+    pass
+
+
+class Unauthorized(Exception):
+    pass
+
+def _basic_auth_str(username, password):
+    return 'Basic ' + to_native_string(
+        b64encode(('%s:%s' % (username, password)).encode('latin1')).strip()
+    )
+
+def _headers(username, password):
+    auth = _basic_auth_str(username, password)
+    return {'Authorization': auth, 'Content-Type': 'application/JSON'}
+
+def get(resource_group_name,registry_name,name,api_username,api_password):
+    registry_name, _ = get_registry_by_name(registry_name, resource_group_name)
+    id=registry_name.id
+    Endpoint = "http://api20170808022712.azurewebsites.net/multi"+id+"/buildDefinitions/"+name
+    response = requests.get(Endpoint,headers=_headers(api_username,api_password))
+    if response.status_code==200:
+        return response.json()
+    raise CLIError("There was an error retrieving the resource. " + str(response.status_code))
+    
+def put(data,username,password,registry_name,name,resource_group_name):
+    registry_name, _ = get_registry_by_name(registry_name, resource_group_name)
+    id=registry_name.id
+    Endpoint = "http://api20170808022712.azurewebsites.net/multi"+id+"/buildDefinitions/"+name
+    response = requests.put(Endpoint,json=data,headers=_headers(username,password))
+    if response.status_code==200:
+        return 'Build definition for multi-architecture manifest successfully created.'
+    raise CLIError("There was an error creating the resource. " + str(response.status_code))
+
+def delete(resource_group_name,registry_name,name,api_username,api_password):
+    registry_name, _ = get_registry_by_name(registry_name, resource_group_name)
+    id=registry_name.id
+    Endpoint = "http://api20170808022712.azurewebsites.net/multi"+id+"/buildDefinitions/"+name
+    response = requests.delete(Endpoint,headers=_headers(api_username,api_password))
+    if response.status_code==200:
+        return 'Build definition for multi-architecture manifest successfully deleted.'
+    raise CLIError("There was an error deleting the resource. " + str(response.status_code))
+
```

Comment:
Rename the above methods (`get`, `put`, `delete`) to something more descriptive.